RecoTauTag/RecoTau/BuildFile.xml

```diff
@@ -24,6 +24,8 @@
 <use   name="TrackingTools/TrackAssociator"/>
 <use   name="PhysicsTools/JetMCUtils"/>
 <use   name="CommonTools/Utils"/>
+<use   name="FastSimulation/BaseParticlePropagator"/>
```

Comment:
Hi Slava, all,

I understand that you have raised this issue for Chris/David et al. So I only add a few sentences for further clarification of the use of the class here: we need this for the calculation of the famous variable "phiAtEcalEntrance" as input to the anti-electron MVA discriminator. In case that this variable is re-calculated from miniAOD event content we have to re-construct it from a transient track propagation up to the ECAL surface.

Cheers,
Roger